tests/utils/googleAnalytics/click.test.ts

```diff
@@ -4,17 +4,10 @@ import * as clickUtils from '../../../src/utils/googleAnalytics/click';
 
 describe('googleAnalytics.click', () => {
   const setUp = () => {
-    /**
-     *  Suppose GA is initialized
-     */
-    if (!window.dataLayer || !Array.isArray(window.dataLayer)) {
-      window.dataLayer = window.dataLayer ?? [];
-    }
-
     const name = faker.lorem.word();
     const params = {foo: 'bar'};
-    const gtagSpy = jest.spyOn(initUtils, 'gtag');
-    const consoleInfoSpy = jest.spyOn(console, 'info');
+    const gtagSpy = jest.spyOn(initUtils, 'gtag').mockImplementation(() => null);
+    const consoleInfoSpy = jest.spyOn(global.console, 'info');
```

Comment:
```suggestion
    const consoleInfoSpy = jest.spyOn(console, 'info');
```
CI 실패 ㅠㅠ global 제거해주면 해결될 것 같습니다 ㅎㅎ

Reply:
실패했네요 ㅠㅠ 감사합니다 ! 다시 해서 올릴게요 ~!